### ISSUE TITLE
Lighthouse improvements

### ISSUE THREAD

**Issue by 1ctinus** [sergeant-savage/chatter#40]
I believe that these improvements are trivial to implement and can improve SEO.
![image](https://github.com/sergeant-savage/chatter/assets/72636637/ac2269f9-3cb3-4f03-900d-3733aa55da29)

- Add a meta tag
- Change the link color
- Add button descriptions
- "Links do not have descriptive text"
I believe this is low priority, but is important in the long run.

**Comment by 1ctinus** [comment]
har har har h har har

**Comment by sergeant-savage** [comment]
These can wait until after #44 is completed, as it would be redundant to do them twice.
